docs/oras-auth-provider.md

```diff
@@ -0,0 +1,144 @@
+5. Install Mutating Admission Webhook onto AKS cluster by following steps [here](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
+6. As the guide linked above shows, it's possible to use the AZ workload identity CLI or the regular az CLI to perform remaining setup. Following steps follow the AZ CLI.
+7. Create ACR AAD application: `az ad sp create-for-rbac --name "<APPLICATION_NAME>"`
+8. On Portal or AZ CLI, enable acrpull role to the AAD application for the ACR resource
+9. Use kubectl to add service account to cluster: 
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    azure.workload.identity/client-id: ${APPLICATION_CLIENT_ID}
+  labels:
+    azure.workload.identity/use: "true"
+  name: ${SERVICE_ACCOUNT_NAME}
+  namespace: ${SERVICE_ACCOUNT_NAMESPACE}
+EOF
+```
+10. From Azure Cloud Shell: 
+```
+export APPLICATION_OBJECT_ID="$(az ad app show --id ${APPLICATION_CLIENT_ID} --query objectId -otsv)"
+cat <<EOF > body.json
+{
+  "name": "kubernetes-federated-credential",
+  "issuer": "${SERVICE_ACCOUNT_ISSUER}",
+  "subject": "system:serviceaccount:${SERVICE_ACCOUNT_NAMESPACE}:${SERVICE_ACCOUNT_NAME}",
+  "description": "Kubernetes service account federated credential",
+  "audiences": [
+    "api://AzureADTokenExchange"
+  ]
+}
+EOF
+
+az rest --method POST --uri "https://graph.microsoft.com/beta/applications/${APPLICATION_OBJECT_ID}/federatedIdentityCredentials" --body @body.json
+```
+11. In the Pod spec, add the `serviceAccountName` property. Example Pod spec:
+```
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: quick-start
+  namespace: <SERVICE_ACCOUNT_NAMESPACE>
+spec:
+  serviceAccountName: <SERVICE_ACCOUNT_NAME>
+  ...
+EOF
+```
+
+#### Ratify Auth Provider Configuration
+```
+"auth-provider": {
+    "name": "azure-wi"
+}
+```
+
+### 3. Kubernetes Secrets
+Ratify resolves registry credentials from Docker Config Kubernetes secrets in the cluster. Ratify considers kubernetes secrets in two ways. First, the configuration can specify a list of `secrets`. Each entry requires the `secretName` field. The `namespace` field must also be provided if the secret does not exist in the namespace ratify is deployed in. Second, Ratify considers the `imagePullSecrets` specified in the service account associated with Ratify. The  `serviceAccountName` field specifies the service account associated with Ratify.
+
+Ratify only supports the kubernetes.io/dockerconfigjson secret type or the legacy kubernetes.io/dockercfg type.  
+
+#### Sample Configuration
+```
+"auth-provider": {
+    "name": "k8s-secrets",
+    "serviceAccountName": "ratify-sa", // will be 'default' if not specified
+    "secrets" : [
+        {
+            "secretName": "artifact-pull-docker-config" // ratify namespace will be used 
+        },
+        {
+            "secretName": "artifact-pull-docker-config2",
```

Comment:
There are some assumptions in this section and might help clarifying them, the service account and the required RBAC for querying the secrets, do you see benefit in adding details of clarify it and its configuration? Or you already have them in the other sections of this document, we can also refer to them using hyper links